tests/test_sqlalchemy_athena.py

```diff
@@ -1331,3 +1331,35 @@ def test_create_table_with_date_partition(self, engine):
         assert tblproperties["projection.dt.type"] == "date"
         assert tblproperties["projection.dt.range"] == "NOW-1YEARS,NOW"
         assert tblproperties["projection.dt.format"] == "yyyy-MM-dd"
+
+    def test_insert_from_select_cte_follows_insert_one(self, engine):
+        engine, conn = engine
+        dialect = engine.dialect
+        metadata = MetaData(schema=ENV.schema)
+        my_table = Table('mytable', metadata,
+              Column('myid', Integer),
+              Column('name', String(30)),
+              Column('description', String(30)))
+        my_other_table = Table('myothertable', metadata,
+              Column('otherid', Integer, primary_key=True),
+              Column('othername', String(30)))
+
+
+        cte = select([my_table.c.name]).where(my_table.c.name == 'bar').cte()
+
+        sel = select([my_table.c.myid, my_table.c.name]).where(
+            my_table.c.name == cte.c.name)
+
+        ins = my_other_table.insert().\
+            from_select(("otherid", "othername"), sel)
+        self.assert_compile(
```

Comment:
The method assert_compile seems to be specific to the SQLAlchemy test class, so it needs to be changed to use assert, which Python supports by default.